port/common/omrthread.c

```diff
@@ -0,0 +1,41 @@
+#if defined (LINUX)
```

Comment:
if this is specific to linux, it needs to be in the linux dir not common

Reply:
youll still have to add a common implementation, even if all it does is return an error